src/test/java/org/opensearch/neuralsearch/query/HybridQueryBuilderTests.java

```diff
@@ -596,6 +597,88 @@ public void testRewrite_whenMultipleSubQueries_thenReturnBuilderForEachSubQuery(
         assertEquals(termSubQuery.value(), termQueryBuilder.value());
     }
 
+    @SneakyThrows
+    public void testBoost_whenNonDefaultBoostSet_thenFail() {
+        // create query with 6 sub-queries, which is more than current max allowed
+        XContentBuilder xContentBuilderWithNonDefaultBoost = XContentFactory.jsonBuilder()
```

Comment:
Nit. Could be nice to have it as json format in comment?

Reply:
ack